docker/Dockerfile

```diff
@@ -60,11 +60,11 @@ RUN apt-get install yarn -y
 ARG SERVER_BRANCH=master
 ARG WEB_APP_BRANCH=master
```

Comment:
We don't have a `master` branch... I know this is changed based on the current build, but the default should be `main` maybe.

Reply:
Thanks, changed it. Although I don't think this is actually doing anything since we normally build from a different branch or tag and it seems to work fine. 🤷‍♂️